include/event_logger.hpp

```diff
@@ -105,6 +103,34 @@ class EventLogger
         const std::optional<std::pair<std::string, std::string>> i_userData1,
         const std::optional<std::pair<std::string, std::string>> i_userData2);
 
+    /**
+     * @brief An API to create a PEL without any callout.
+     *
+     * @param[in] i_errorType - Enum to map with event message name.
+     * @param[in] i_severity - Severity of the event.
+     * @param[in] i_fileName - File name.
+     * @param[in] i_funcName - Function name.
+     * @param[in] i_internalRc - Internal return code.
+     * @param[in] i_description - Error description.
+     * @param[in] i_userData1 - Additional user data [optional].
+     * @param[in] i_userData2 - Additional user data [optional].
+     * @param[in] i_symFru - Symblolic FRU callout data [optional].
+     * @param[in] i_procedure - Procedure callout data [optional].
+     *
+     * @todo: Symbolic FRU and procedure callout needs to be handled in this
+     * API.
+     */
+    static void createAsyncPel(const types::ErrorType& i_errorType,
```

Comment:
Why not name like other APIs
createAsyncPelWithoutInventoryCallout?

Reply:
other API's are representing "WithInventoryCallout" , "WithI2cDeviceCallout" ,  "WithI2cBusCallout"
this API is not taking any of these input for callouts. So kept the simple name.

Reply:
Then why the brief gives that information explicitly? 
So if this API is explicitly meant to log PEL without call out, why the name should not represent that?

Reply:
updated the doxygen.